spec/support/helpers/feature_helpers.rb

```diff
@@ -34,27 +34,47 @@ def seen_pages
     end
   end
 
-  def answer_gyr_triage_questions(
-    screenshot_method: nil,
-    income_level: "hh_1_to_25100_html",
-    id_type: "have_paperwork",
-    doc_type: "all_copies_html",
-    filed_past_years: [
-      TaxReturn.current_tax_year - 3,
-      TaxReturn.current_tax_year - 2,
-      TaxReturn.current_tax_year - 1,
-    ],
-    assistance_options: ['chat'],
-    income_type_options: ['none_of_the_above']
-  )
+  class TriageChoices
+    def initialize(choices)
+      @choices = choices
+    end
+
+    private
+
+    def method_missing(symbol, *args)
```

Comment:
If you're gonna support `method_missing`, why switch to `"#{choices.doc_type"`? Mostly I'm confused; not a blocker.

Reply:
my goal was to convert the simple list of keyword params for `answer_gyr_triage_questions` into an object such that every test in `features/triage_spec.rb` is forced to provide every single piece of information that will be used.

prior to this, any missing kwargs would revert to the default argument, which is (arguably) confusing because you have to look in two places to find the exact options that will be used

but the `method_missing` thing isn't the only way to do it. now it has the new problem that you could initialize `TriageChoices` with an option that doesn't exist and it wouldn't complain, unlike the kwargs. I mostly also wanted to keep it concise.

Reply:
I get it now [hi this is asheesh] `choices = TriageOptions.new(...)`